lib/grp.gi

```diff
@@ -4673,6 +4674,14 @@ InstallMethod( MinimalNormalSubgroups,
     function( G )
     local nt, c, r, U;
 
+    # force an IsNilpotent check
+    # should have and IsSolvable check, as well,
+    # but methods for solvable groups are only in CRISP
+    # which aggeressively checks for solvability, anyway
+    if (not HasIsNilpotentGroup(G) and IsNilpotentGroup(G)) then
+      return MinimalNormalSubgroups( G );
```

Comment:
@fingolfin Actually, these two lines will never run if CRISP is loaded, because CRISP has a redispatch for finiteness and solvability checking with a higher rank, and then the CRISP method will be chosen. Nevertheless, I added a test for ````Socle```` and one for ````MinimalNormalSubgroups````, and they are supposed to trigger the codelines in question if CRISP is not loaded.